src/clique/CliqueManager.cc

```diff
@@ -266,12 +266,12 @@ void CliqueManager::SetByteLimits()
   m_allReduceByteLimit = rcclParamAllReduceCliqueByteLimit();
   if (m_allReduceByteLimit == 0)
   {
-    switch (m_gcnArch)
-    {
-    case 906: m_allReduceByteLimit =  16777216; break;
-    case 908: m_allReduceByteLimit =   8388608; break;
-    default:  m_allReduceByteLimit =  16777216; break;
-    }
+    if (strncmp(m_gcnArchName, "gfx906", 6) == 0)
+      m_allReduceByteLimit = 16777216; // TODO: do we really need this, since it's covered by the default?  Check if it's more of a documentation / assumption thing.
```

Comment:
Clique isn't used anymore, but this was done explicitly so that it's easy to tell that it had been "optimized" for 906.
Please leave it in.